benchmark/benchmark_serving.py

```diff
@@ -131,7 +151,14 @@ def main(args: argparse.Namespace):
         "--tokenizer", type=str, required=True, help="Name or path of the tokenizer."
     )
     parser.add_argument(
-        "--num-prompts", type=int, default=100, help="Number of prompts to process."
+        "--num-prompts", type=int, default=1000, help="Number of prompts to process."
```

Comment:
Is it possible to set it to 100 and concurrency to 100 to make it compatible with old behaviour?

Reply:
ok